monitoring/mock_uss/geoawareness/parsers/ed269/__init__.py

```diff
@@ -87,10 +88,28 @@ class WeekDateType(str, Enum):
     ANY = "ANY"
 
 
+class ED269TimeType(str):
+    """String that allows values which describe a time in ED-269 flavour of ISO 86001 format.
+
+    ED-269 standard specifies that a time instant type should be in the form of hh:mmS where S is
+    the timezone. However, examples are using the following format: 00:00:00.00Z
+    This class supports both formats.
+    """
+
+    def __new__(cls, value):
```

Comment:
I know `implicitdict` doesn't do this right now, but it would probably be useful to type-hint: `value: Union[str, time]` (or whatever the appropriate types are)